### DISCUSSION TITLE
출시 관련 문제

### DISCUSSION THREAD

**Discussion by kihye** [naverz/zepeto-studio-kor#635] [Issue Report (Legacy)]
월드 출시 후에 수정사항 반영해서 업데이트 하려고 크롬으로 월드 콘솔 접속후 새 버전 추가 시도, 이후 나오는 팝업에서 확인까지 눌렀는데 아예 반응이 없네요...엣지로도 접속해서 위와 같은 방법을 반복해봤고, 다른 컴퓨터에서도 동일한 방법을 시도해봤는데 역시 무반응이었습니다.

이 부분 한번 확인해주실 수 있나요?

**Comment by kihye** [comment]
출시 완료는 어제인 4월 18일 12:16 AM에 됐습니다. 혹시 출시 완료 시간대와 상관관계가 있는걸까요...

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, 출시 관련된 문의는 고객센터쪽에 해보시면 정확한 로그와 함께 확인이 가능하실 것으로 보입니다.
https://support.zepeto.me/hc/ko

**Reply by kihye** [reply]
넵 그러면 그쪽으로 문의 넣어보겠습니다. 감사합니다.
